hvplot/tests/testutil.py

```diff
@@ -6,6 +6,15 @@
 import pandas as pd
 import panel as pn
 import pytest
+import dask.dataframe as dd
+import spatialpandas as spd
+
+try:
+    import spatialpandas
+    import dask
+except ImportError:
+    spatialpandas = None
+    dask = None
```

Comment:
Sorry for the late re-review. I think these should be consolidated into a single try/except, or else it'll crash on import if dask/spatialpandas is unavailable.
```suggestion
import dask.dataframe as dd
import spatialpandas as spd

try:
    import spatialpandas
    import dask
except ImportError:
    spatialpandas = None
    dask = None
```

Reply:
Done!